spec/build/script/csharp_spec.rb

```diff
@@ -13,6 +13,7 @@
 
   describe 'configure' do
     it 'sets up package repository' do
+      data[:config][:os] = 'linux'
```

Comment:
Ok, so it's just the unit test environment that doesn't? If the unit tests set it too, we should revert these changes to ensure we're as close to the real world as possible ;)